src/pypendency/exceptions.py

```diff
@@ -36,3 +39,9 @@ class ServiceInstantiationFailed(Exception):
     def __init__(self, service_fqn: str) -> None:
         self.service_fqn = service_fqn
         super().__init__(f"Type {service_fqn} cannot be instantiated by the container")
+
+
+class TagNotFoundInContainer(Exception):
```

Comment:
Add Exception suffix

Reply:
Other exceptions don't have it (and we shouldn't change their names because it would break the API). Do we want to add the suffix anyways?

Reply:
As discussed, this name will be kept in order to follow the trend